README.md

```diff
@@ -237,3 +237,17 @@ Methylation: a DNA modification where methyl groups of are added to cytosines in
 Molecular aberration type: encapsulates the different changes that occur the DNA, RNA and other biomolecules during cancer. For example: CNAs, SNVs, methylation, mRNA abundance, miRNA abundance.
 
 SNV: Single Nucleotide Variant. a nucleotide base (A,C,G,T) change in DNA.
+
+## License
+
+Author: Natalie Fox, Paul C. Boutros(pboutros@mednet.ucla.edu)
+
+iSubGen is licensed under the GNU General Public License version 2. See the file LICENSE.md for the terms of the GNU GPL license.
+
+iSubGen is a multi-data type subtyping tool, which used intermediary assessments created with autoencoders and similarity calculations.
+
+Copyright (C) 2024 University of California Los Angeles ("Boutros Lab") All rights reserved.
```

Comment:
**(style|non-blocking)** It would be nice to use a range (you can check the commit history)